packages/augur-ui/src/modules/app/store/app-status-hooks.ts

```diff
@@ -710,7 +710,10 @@ export const useAppStatus = (defaultState = DEFAULT_APP_STATUS) => {
         dispatch({ type: SET_CATEGORY_STATS, categoryStats }),
       updateFilterSortOptions: filterSortOptions =>
         dispatch({ type: UPDATE_FILTER_SORT_OPTIONS, filterSortOptions }),
-      setModal: modal => dispatch({ type: SET_MODAL, modal }),
+      setModal: modal => {
+        dispatch({ type: CLOSE_MODAL });
```

Comment:
we shouldn't need to close modal before setting modal, setModal overwrites whatever's already set for modal with whatever you pass it.